fs/accounting/stats_groups.go

```diff
@@ -349,3 +378,26 @@ func (sg *statsGroups) reset() {
 	sg.m = make(map[string]*StatsInfo)
 	sg.order = nil
 }
+
+// delete removes all references to the group.
+func (sg *statsGroups) delete(group string) {
+	sg.mu.Lock()
+	defer sg.mu.Unlock()
+	stats := sg.m[group]
+	if stats == nil {
+		return
+	}
+	stats.ResetErrors()
```

Comment:
Is that needed since we remove it anyway?

Reply:
It is used to set references to nil.